dcos/marathon.py

```diff
@@ -388,6 +388,37 @@ def scale_app(self, app_id, instances, force=None):
         deployment = response.json()['deploymentId']
         return deployment
 
+    def scale_group(self, group_id, scale_factor, force=None):
+        """Scales a group with the requested scale-factor.
+        :param group_id: the ID of the group to scale
+        :type group_id: str
+        :param scale_factor: the requested value of scale-factor
+        :type scale_factor: str
+        :param force: whether to override running deployments
+        :type force: bool
+        :returns: the resulting deployment ID
+        :rtype: bool
+        """
+
+        group_id = self.normalize_app_id(group_id)
+
+        if not force:
+            params = None
+        else:
+            params = {'force': 'true'}
+
+        url = self._create_url('v2/groups{}'.format(group_id))
+
+        scale_factor = util.parse_float(scale_factor)
```

Comment:
To match the rest of the code let's move this to `dcoscli.marathon.main.__group_scale`. This method `scale_group` should accept and assume that `scale_factor` is a `float`. Let's change the pydoc above to reflect that: `type scale_factor: float`.